codice/ddf
# DDF-3811 Pulled embedded query metacards out of their workspace metacard

#### What does this PR do?
 - Updates Workspace metacards to no longer have query metacards embedded within them.
 - Adds a new model type on the frontend to handle partial models.  This should help in the future when we go to pull Lists out from workspaces as well.

#### How should this be tested?
Test all of the different paths for workspaces.  Verify that you can create, modify, delete workspaces without issues.  Verify that local workspaces are also unaffected.

#### Any background context you want to provide?
#### What are the relevant tickets?
[DDF-3811](https://codice.atlassian.net/browse/DDF-3811)

#### Checklist:
